x-pack/plugins/log_explorer/common/datasets/models/dataset.ts

```diff
@@ -86,6 +86,9 @@ export class Dataset {
     );
 
     // Create new datasets for the retrieved wildcards
-    return Array.from(wildcards).map((wildcard) => Dataset.create({ name: wildcard }));
+    return Array.from(wildcards).map((wildcard) => {
+      const [, dataset] = wildcard.split('-');
```

Comment:
**nit:** It might be useful to apply this fallback logic for every Dataset, so moving this logic to the .create factory function, where it can set the title equal to this dataset name if one is not provided.